package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
```

Comment:
this file is deleted

Reply:
Oops, forgot about that reference. Should this point to the `cli.js` now instead?

Reply:
no, generally just remove it since there isn't an API. having it be the CLI would be an anti-pattern since it would run side effects and expect to be the entrypoint of the CLI

Reply:
Would it make sense to create a `src/lib/index.js` to import all of the other files under `lib`, and use that as the main entrypoint in the API?

Reply:
only if you intend people to use that directly, not if it is just for tests

Reply:
> yeah, if you want this to be consumed by another npm package (e.g. a github app) you might want to have `cli.js` consume the parameters from the cli and shim over to a js file that takes an options object and does the work. And then you make your main point to that file.

Hmmm yeah that makes a lot of sense. Eventually I'd like to implement a library such as `oclif`, and I think it would make a lot of sense to restructure like that.